dbt_platform_helper/providers/files.py

```diff
@@ -24,3 +24,8 @@ def mkfile(base_path: str, file_path: str, contents: str, overwrite=False) -> st
         file.write_text(contents)
 
         return f"File {file_path} {action}"
+
+    @staticmethod
+    def delete_file_if_present(base_path: str, file_path: str | Path):
```

Comment:
Is there ever a use case where we would want it to error if the file doesn't exist?

If we did, I think we might check for it's existence separately.

Would this be enough for the name?
```suggestion
    def delete_file(base_path: str, file_path: str | Path):
```